### DISCUSSION TITLE
Add javascript

### DISCUSSION THREAD

**Discussion by Modrx** [mastodon/mastodon#28890] [Q&A]
Hi,

How can I add javascript tags to the header, like this:
`<script></script>`

Thank you!

**Comment by renchap** [comment]
This is not supported by Mastodon directly, you will need to patch the source code to achieve this.

**Comment by Modrx** [comment]
I understand, can you please tell me what file I need to edit and what code to add to link to external javascript file or how to load javascript code in the header within the <script></script> tag.

Thank you!
